bcda/bcdacli/cli_test.go

```diff
@@ -836,3 +834,110 @@ func getRandomPort(t *testing.T) int {
 
 	return listener.Addr().(*net.TCPAddr).Port
 }
+
+func CreateTestZipFile(file string) error {
+	zf, err := os.Create(file)
+	if err != nil {
+		return err
+	}
+	defer zf.Close()
+
+	w := zip.NewWriter(zf)
+	var files = []string{
+		"T.BCD.A0001.ZC8Y18.D181120.T1000001",
+		"T.BCD.A0001.ZC8Y18.D181120.T1000002",
+		"T.BCD.A0001.ZC8Y18.D181120.T1000003",
+	}
+
+	for _, file := range files {
+		f, err := w.Create(file)
+		if err != nil {
+			return err
+		}
+		_, err = f.Write([]byte("foo bar"))
+		if err != nil {
+			return err
+		}
+	}
+
+	return w.Close()
+}
+
+func getFileCount(path string) int {
+	f, _ := ioutil.ReadDir(path)
+	return len(f)
+}
+
+func TestCloneCCLFZips(t *testing.T) {
+	srcZips := []string{
+		"T.BCD.A0002.ZCY18.D181120.T9999990",
+		"T.BCD.A0002.ZCY18.D181120.T9999991",
+		"T.BCD.A0002.ZCY18.D181120.T9999992",
+		"not_a_cclf_file",
+	}
+
+	for _, srcZip := range srcZips {
+		err := CreateTestZipFile(srcZip)
+		if err != nil {
+			t.Fatal(err)
+		}
+		defer func(f string) { os.Remove(f) }(srcZip)
```

Comment:
May want to consider creating a temporary directory (https://golang.org/pkg/io/ioutil/#TempDir) and putting all of these files underneath the temp dir.

So at the beginning you can do something like this.
```
dir, err := ioutil.TempDir("", "clone_cclf")
	if err != nil {
		log.Fatal(err)
	}
defer os.RemoveAll(dir)
```

So worse comes to worse, we fail to clean up the temporary directory. If you run these tests locally, it'll create these files in bcda-app, which would have to be manually cleaned up.
```

Reply:
This was a great idea. Made clean up much simpler and much safer now that I only have to manage cleaning up one folder and not a bunch of files.